database/migrations/2024_12_20_144244_create_tasks_table.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    public function up(): void
+    {
+        Schema::create('tasks', function (Blueprint $table) {
+            $table->id();
+            $table->string('title');
+            $table->string('description');
+            $table->enum('status', ['Pending', 'In Progress', 'Completed'])->default('Pending');
```

Comment:
You could use an Enum class let's say TaskStatus so it's easier to manage and also to make tests later
```
<?php

declare(strict_types=1);

namespace Domain\Resources\Enums;

enum TaskStatus: string
{
    case Pending = 'pending';
    case InProgress = 'in_progress';
    case Completed = 'completed';
}
```

Reply:
Also, just saw that you used enum
https://lightit.slite.page/p/-BOPevWbGOt-U1/Untitled#9d6a2b09